cfme/tests/test_utilization.py

```diff
@@ -9,7 +12,14 @@
 from cfme.common.provider import BaseProvider
 from cfme.exceptions import FlashMessageException
 
-pytest_generate_tests = testgen.generate(testgen.provider_by_type, None)
+
+# Tests for vmware and rhev providers have been moved to cfme/tests/test_utilization_metrics.py.
+# Also, this test just verifies that C&U/perf data is being collected, whereas the tests in
+# test_utilization_metrics.py go a step further and verify that specific performance metrics are
+# being collected.Eventually, support should be added to verify that specific metrics are being
+# collected for *all* providers.
+pytest_generate_tests = testgen.generate(testgen.provider_by_type, ['openstack', 'ec2',
```

Comment:
`testgen.provider_by_type` also accepts provider "categories" so you probably want to use `['cloud', 'container', 'middleware']` instead of listing specific provider types.